packages/react-static/src/static/webpack/rules/cssLoader.js

```diff
@@ -17,13 +17,14 @@ function initCSSLoader() {
         // Necessary for external CSS imports to work
         // https://github.com/facebookincubator/create-react-app/issues/2677
         sourceMap: true,
-        ident: 'postcss',
```

Comment:
This is no longer needed

https://github.com/webpack-contrib/postcss-loader/blob/master/CHANGELOG.md#notes

> - you don't need ident option for loader